geocity/apps/permits/services.py

```diff
@@ -1907,6 +1913,10 @@ def get_works_type_names_list(permit_request):
     return permit_request.get_works_type_names_list()
 
 
+def get_works_object_names_list(permit_request):
+    return permit_request.get_works_object_names_list()
+
+
```

Comment:
Please remove. If you can pass `permit_request `to this method, you can call directly the method on the `permit_request `instance

Reply:
So also for `def get_works_type_names_list(permit_request):` I guess?